### PR TITLE
[DOCS]: Add role to the snowflake Gem documentation

### PR DESCRIPTION
**Pull request by kishoreypro**
Asana: https://app.asana.com/0/1202124936707798/1204423127241985/f

### CODE REVIEW

**Review by nadeesh-garg — APPROVED** [review]

File: docs/low-code-spark/gems/source-target/warehouse/snowflake.md [hunk 1]
```diff
@@ -157,33 +133,10 @@ object sf_customer {
...
-  {
-    "image":"/img/snowflake/write/5.png",
-    "description":<h3 style={{padding:'10px'}}>Step 5 - Define 'Write Mode' and optionally provide 'Post-Script SQL'</h3>,
-  }
-];
-
-<App ImageData={ImageData2}></App>
-```
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
```
[the remark this answers is not in the document]
Comment (kishoreypro) [reply docs/low-code-spark/gems/source-target/warehouse/snowflake.md:138]: Changing the example from image to video

File: docs/low-code-spark/gems/source-target/warehouse/snowflake.md [hunk 2]
```diff
@@ -141,8 +116,9 @@ object sf_customer {
...
-| Table           | The name of the table to which data is to be written.                                                                                                                                                                                                                                                                     | True                                                   |
+| Warehouse       | Default virtual warehouse to use for the session after connecting                                                                                                                                                                                                                                                         | False                                                  |
+| Role            | Default security role to use for the session after connecting                                                                                                                                                                                                                                                             | False                                                  |
```
[the remark this answers is not in the document]
Comment (kishoreypro) [reply docs/low-code-spark/gems/source-target/warehouse/snowflake.md:121]: Default values are set by the snowflake admins.. it keep changing from org to org.. I don't think we set the default values

File: docs/low-code-spark/gems/source-target/warehouse/snowflake.md [hunk 3]
```diff
@@ -31,10 +31,11 @@ Please refer the snowflake official documentation to attach private key to user
...
-| Table                  | The name of the table to be read. All columns and records are retrieved (i.e. it is equivalent to `SELECT * FROM table`).                                 | Required if `Data Source` is `DB Table`                |
-| SQL Query              | The exact query (`SELECT` statement) to run                                                                                                               | Required if `Data Source` is `SQL Query`               |
+| Table                  | Name of the table to be read. All columns and records are retrieved (i.e. it is equivalent to `SELECT * FROM table`).                                     | Required if `Data Source` is `DB Table`                |
+| SQL Query              | Exact query (`SELECT` statement) to run                                                                                                                   | Required if `Data Source` is `SQL Query`               |
```
Comment (nadeesh-garg) [inline docs/low-code-spark/gems/source-target/warehouse/snowflake.md:38]: Can you add if the statement can ONLY be a Select Statement or can one run anything?

Comment (kishoreypro) [reply docs/low-code-spark/gems/source-target/warehouse/snowflake.md:38]: it can be only select statement only

**Review by nadeesh-garg — APPROVED** [review]

**Pull request merged**